### PR TITLE
Fix the error documented to report an unvetted package

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
This has been true since 2.4.0 but apparently never fixed in the documentation.

### CODE REVIEW

**Comment by stefanobaghino-da** [comment]
Error on CI which seems very unlikely to have been caused by this PR:
```
Running Sphinx v4.3.1
loading translations [en]... done

Sphinx version error:
The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.
```

**Comment by garyverhaegen-da** [comment]
https://github.com/digital-asset/docs.daml.com/pull/614 should fix that.

**Comment by stefanobaghino-da** [comment]
I'll wait for that to be merged then, thank you!

File: docs/2.7.6/docs/canton/usermanual/packagemanagement.rst [hunk 1]
```diff
@@ -66,7 +66,7 @@ synchronise on a transaction that uses packages contained in a certain DAR, we n
 have uploaded the same DAR before the transaction was submitted.
 
 If one of the involved participants doesn't know about a certain DAR, then the transaction will bounce with an error
-PACKAGE_NOT_VETTED_BY_RECIPIENTS.
+``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing what package has not been vetted on which participant).
```
Comment (carrielaben-da) [inline docs/2.7.6/docs/canton/usermanual/packagemanagement.rst:69]:
```suggestion
``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing which package has not been vetted on which participant).
```

File: docs/2.8.0/docs/canton/usermanual/packagemanagement.rst [hunk 2]
```diff
@@ -66,7 +66,7 @@ synchronise on a transaction that uses packages contained in a certain DAR, we n
 have uploaded the same DAR before the transaction was submitted.
 
 If one of the involved participants doesn't know about a certain DAR, then the transaction will bounce with an error
-PACKAGE_NOT_VETTED_BY_RECIPIENTS.
+``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing what package has not been vetted on which participant).
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/canton/usermanual/packagemanagement.rst:69]:
```suggestion
``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing which package has not been vetted on which participant).
```

File: docs/2.9.0/docs/canton/usermanual/packagemanagement.rst [hunk 3]
```diff
@@ -66,7 +66,7 @@ synchronise on a transaction that uses packages contained in a certain DAR, we n
 have uploaded the same DAR before the transaction was submitted.
 
 If one of the involved participants doesn't know about a certain DAR, then the transaction will bounce with an error
-PACKAGE_NOT_VETTED_BY_RECIPIENTS.
+``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing what package has not been vetted on which participant).
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/canton/usermanual/packagemanagement.rst:69]:
```suggestion
``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing which package has not been vetted on which participant).
```

**Review by carrielaben-da — APPROVED** [review]
Slight reword.

**Pull request merged**